src/pages/packet/components/packet-row.jsx

```diff
@@ -0,0 +1,30 @@
+import PacketStruct from '../../../data/packet-structure.json';
+import PackageComponent from './package-component';
+import React from 'react';
+
+const PackageRow = ({ trail, build, engage, recruit, TRAILS_LIST, removeOptionChecked, setTrail, isMobile }) => {
+  const row = []
+  const selectedLevel = trail === "build" ? build : trail === "engage" ? engage : recruit
+  PacketStruct["trails"][trail].forEach((level, index) => {
+    row.push(<PackageComponent className={`row-${trail}-${level}`} trail={trail} level={index + 1} key={`${trail}-${level}`}
+                               isMobile={isMobile} perks={level} callback={(level, trailType) => setTrail(level, trailType)}
+                               selected={selectedLevel === index + 1}/>)
+  })
+
+  return (
```

Comment:
if it's easier/the two are different enough for this one you can do if isMobile return <> else return <>

but this works too so up to you